File: 백준/Bronze/28113. 정보섬의 대중교통/정보섬의 대중교통.cs
using System;

class Q28113 {
    static void Main(string[] args)
    {
        string[] S = Console.ReadLine().Split();
        int N = int.Parse(S[0]);
        int A = int.Parse(S[1]);
        int B = int.Parse(S[2]);
        
        if (N <= B && B == A)
        {
			Console.WriteLine("Anything");
		}
        else if (N <= B && B - A > 0)
        {
			Console.WriteLine("Bus");
		}
        else
        {
			Console.WriteLine("Subway");
		}
    }
}

File: 백준/Bronze/28113. 정보섬의 대중교통/정보섬의 대중교통.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int N = sc.nextInt();
        int A = sc.nextInt();
        int B = sc.nextInt();
        
        if (N <= B && B == A) {
			System.out.println("Anything");
		} else if (N <= B && B - A > 0) {
			System.out.println("Bus");
		} else {
			System.out.println("Subway");
		}
    }
}